File: Application/App/src/app/models/matchDetails/team-dto.ts
import { BanDto } from "./ban-dto";
import { ObjectivesDto } from "./objectives-dto";

export class TeamDto {
    bans : BanDto[] = [];
    objectives : ObjectivesDto = new ObjectivesDto({});
    teamid : number = 0;
    win : boolean = true;

    public constructor(data : Partial<TeamDto>){
        Object.assign(this,data);
    }
}

File: Backend/Backend/Models/Matches/TeamDto.cs
namespace Backend.Models.Matches
{
    public class TeamDto
    {
        public TeamDto(List<BanDto> bans, ObjectivesDto objectives, int teamId, bool win)
        {
            Bans = bans;
            Objectives = objectives;
            TeamId = teamId;
            Win = win;
        }

        public List<BanDto> Bans { get; set; }
        public ObjectivesDto Objectives { get; set; }
        public int TeamId { get; set; }
        public bool Win { get; set; }
    }
}
